lms/djangoapps/bookmarks/api.py

```diff
@@ -0,0 +1,27 @@
+"""
+This File contains bookmark api method(s).
+"""
+
+from bookmarks.serializers import BookmarkSerializer
+from bookmarks.models import Bookmark
+
+
+def get_bookmark(user, usage_key, fields=None, serialized=False):
+    """
+    Return bookmark model or data.
+
+    Args:
+        user (User): The user requesting the bookmark.
+        fields_to_add (list): List of fields to return for a bookmark.
+        usage_key (UsageKey): The usage id of an Xblock.
+        serialized (Bool): Decides to return object or json.
+
+    Returns:
+         A dict or object containing bookmark data.
+
+    Raises:
+         ObjectDoesNotExit: If Bookmark object does not exist.
+    """
+    bookmark = Bookmark.objects.get(usage_key=usage_key, user=user)
```

Comment:
Nit: I find this confusing because it isn't a single bookmark, is it? It returns all the bookmarks in a course for the given user, correct?

I would expect the method to be "get_bookmarks", and the name of the variable to be "bookmarks".